Vagrantfile

```diff
@@ -360,11 +377,13 @@ Vagrant.configure("2") do |config|
           vb.name = hostname
         end
       elsif(vagrant_provider == 'libvirt')
+        host.vm.synced_folder ".", "/vagrant"
```

Comment:
We can remove this, because is the same as the default in vagrant-libvirt

Reply:
fixed thanks!